.github/workflows/release-to-edge.yml

```diff
@@ -0,0 +1,112 @@
+name: Release
+
+on:
+  # Run the workflow each time new commits are pushed to the candidate branch. 
+  push:
+    branches: [ "2204" ]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  issues: write
+
+jobs:
+  get-architectures:
+    name: 🖥 Get snap architectures
+    runs-on: ubuntu-latest
+    outputs:
+      architectures: ${{ steps.get-architectures.outputs.architectures }}
+      architectures-list: ${{ steps.get-architectures.outputs.architectures-list }}
+    steps:
+      - name: 🖥 Get snap architectures
+        id: get-architectures
+        uses: snapcrafters/ci/get-architectures@main
+        with:
+          snapcraft-project-root: "ffmpeg-2204-sdk"
+
+  release-sdk:
+    name: 🚢 Release to latest/edge
+    needs: get-architectures
+    runs-on: ubuntu-latest
+    environment: "2204 Branch"
+    strategy:
+      matrix:
+        architecture: ${{ fromJSON(needs.get-architectures.outputs.architectures-list) }}
+    steps:
+      - name: 🚢 Release to latest/edge
+        uses: snapcrafters/ci/release-to-candidate@main
+        with:
+          architecture: ${{ matrix.architecture }}
+          channel: "edge-${{ github.event.number }}"
+          launchpad-token: ${{ secrets.LP_BUILD_SECRET }}          
+          snapcraft-project-root: "ffmpeg-2204-sdk"
+          store-token: ${{ secrets.SNAP_STORE_CANDIDATE }}
+            
+  update-content-snap:
+    name: Update the content snap
+    needs: [ release-sdk, get-architectures ]
+    environment: "2204 Branch"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update the content snap
+        uses: snapcrafters/ci/sync-version@main
+        with:
+          token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
+          snapcraft-project-root: "ffmpeg-2204"
+          update-script: |
+            sed -i 's/^\(- ffmpeg-2204-sdk\/latest\/\)stable$/\1edge-${{ github.event.number }}/' ffmpeg-2204/snapcraft.yaml
+
+  release-content:
+    name: 🚢 Release to latest/edge
+    needs: get-architectures
+    runs-on: ubuntu-latest
+    environment: "2204 Branch"
+    strategy:
+      matrix:
+        architecture: ${{ fromJSON(needs.get-architectures.outputs.architectures-list) }}
+    steps:
+      - name: 🚢 Release to latest/edge
+        uses: snapcrafters/ci/release-to-candidate@main
+        with:
+          architecture: ${{ matrix.architecture }}
+          channel: "edge-${{ github.event.number }}"
+          launchpad-token: ${{ secrets.LP_BUILD_SECRET }}          
+          snapcraft-project-root: "ffmpeg-2204"
+          store-token: ${{ secrets.SNAP_STORE_CANDIDATE }}
```

Comment:
This won't build the version you just commited. Instead, this will build the version that triggered this workflow. You first need to checkout the latest version of the branch.